tracer/src/Datadog.Trace/Debugger/Snapshots/DebuggerSnapshotCreator.cs

```diff
@@ -689,8 +690,10 @@ internal string FinalizeMethodSnapshot<T>(string probeId, ref CaptureInfo<T> inf
         internal void FinalizeSnapshot(string methodName, string typeFullName, string probeFilePath)
         {
             var activeScope = Tracer.Instance.InternalActiveScope;
-            var traceId = activeScope?.Span?.TraceId.ToString();
-            var spanId = activeScope?.Span?.SpanId.ToString();
+
+            // TODO: support 128-bit trace ids?
```

Comment:
Can you discuss with the DI team please to see if we need to do it?

Reply:
We're adding 128-bit support to tracing only now. Other products will follow.